Failure listing buckets when response is missing Content-Type header

[cyberduck_error.log](https://github.com/user-attachments/files/18906167/cyberduck_error.log)

**Describe the bug**
Error when listing buckets.

**To Reproduce**
Steps to reproduce the behavior:
1. Connect to server does not show the available buckets.

**Desktop (please complete the following information):**
 - Windows 10
 - Cyberduck 9.1.2

**Log Files**
```
Caused by: java.lang.NullPointerException
	at org.jets3t.service.impl.rest.httpclient.RestStorageService.listAllBucketsImpl(RestStorageService.java:1462) ~[Cyberduck.Core.DLL:0.9.38]
	at org.jets3t.service.StorageService.listAllBucketsImpl(StorageService.java:1408) ~[Cyberduck.Core.DLL:0.9.38]
	at org.jets3t.service.StorageService.listAllBuckets(StorageService.java:591) ~[Cyberduck.Core.DLL:0.9.38]
	at org.jets3t.service.S3Service.listAllBuckets(S3Service.java:1446) ~[Cyberduck.Core.DLL:0.9.38]
```

--- a/pom.xml
+++ b/pom.xml
@@ -122,7 +122,7 @@
             <dependency>
                 <groupId>ch.iterate.s3</groupId>
                 <artifactId>jets3t</artifactId>
-                <version>0.9.38</version>
+                <version>0.9.39</version>
             </dependency>
             <dependency>
                 <groupId>com.auth0</groupId>

